### PR TITLE
hari70a add accessibility cheatsheet

### PR DESCRIPTION
**Pull request by Hari70a**
## What does this PR do?
Add accessibility cheatsheet

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Hari70a** [comment]
@davorpa & @eshellman 

Kindly review this one.

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -128,6 +128,7 @@
 
 ### HTML and CSS
 
+* [Accessibility CheatSheet](https://learn-the-web.algonquindesign.ca/topics/accessibility-cheat-sheet/) - Learn the Web, Thomas J Bradley (HTML)
```
Comment (davorpa) [inline more/free-programming-cheatsheets.md:131]: How do you know that Tomas J. Bradley is a creator?

Comment (Hari70a) [reply more/free-programming-cheatsheets.md:131]: @davorpa ,

Based on their github repository

image

Comment (davorpa) [reply more/free-programming-cheatsheets.md:131]: Let's @eshellman decide how to write this

**Comment by Hari70a** [comment]
Mistakenly deleted the head repo.

Once this MR https://github.com/EbookFoundation/free-programming-books/pull/7752 got merged, raise a separate MR for this again

cc: @davorpa & @eshellman

**Pull request closed without merging**